Remotion/Web/Development.WebTesting.UnitTests/Accessibility/AccessibilityConfigurationTest.cs

```diff
@@ -0,0 +1,96 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// 
+using System;
+using NUnit.Framework;
+using Remotion.Web.Development.WebTesting.Accessibility;
+
+namespace Remotion.Web.Development.WebTesting.UnitTests.Accessibility
+{
+  [TestFixture]
+  public class AccessibilityConfigurationTest
+  {
+    [Test]
+    public void GetJSON_FullConfiguration_ValidResult ()
+    {
+      const string expected =
+          "iframes: false, xpath: true, frameWaitTime: 13000, absolutePaths: true, restoreScroll: true, runOnly: {type:\"tag\",values:[\"wcag2a\"]}";
+
+      var configuration = new AccessibilityConfiguration (
+          AccessibilityConformanceLevel.Wcag20_ConformanceLevelA,
+          false,
+          TimeSpan.FromSeconds (13),
+          true,
+          true);
+
+      var result = configuration.GetJson();
```

Comment:
why does the json not have braces?

Reply:
It doesn't return a JSON object, rather the state of the object as "json body content". It's the format the `AccessibilityAnalyzer` needs for injecting. Do we

- [ ] Add braces here and remove them in `Analyzer` (where we need them) ?)
- [ ] Rename the method to `GetJsonContent`
- [ ] None of the above

Reply:
Okay, so is our code the one that doesn't want braces or is that actually what happens inside axe.js? if it's our code, can we change that?

Reply:
Okay, so this is where it's used

```
    private void AppendAxeRunFunctionCallOptions (StringBuilder stringBuilder)
    {
      stringBuilder.Append ('{');
      stringBuilder.Append (Configuration.GetJson());

      if (ExcludedRules.Any())
      {
        stringBuilder.Append (",rules:{");
        stringBuilder.Append (ExcludedRules.Select (x => '"' + x + "\": {enabled: false}").Aggregate ((i, j) => i + ",\n" + j));
        stringBuilder.Append ('}');
      }
    }
```

I suppose it's just our code. If we changed the StringBuilder magic inside `AccessibilityAnalyzer` a bit the config could return a 'complete' json object if so desired

Reply:
Please do. Makes reasoning about the string that much easier.